experiments/AMIP/moist_mpi_earth/coupler_utils/general_helper.jl

```diff
@@ -0,0 +1,20 @@
+# most of these functions are temporary helpers until upstream issues are resolved
+
+# TODO: unify with coupler interface
+struct CouplerSimulation{I, F, B, T, M}
+    Δt::I
+    t::F
+    boundary_space::B
+    FT::T
+    mask::M
+end
+
+get_u(sim, t) = Geometry.UVVector.(sim.integrator.sol.u[t].c.uₕ).components.data.:1
```

Comment:
I'd like this to have a more specific name since `u` could refer to `integrator.sol.u`

Reply:
not being used :) removed it